CMakeLists.txt

```diff
@@ -13,12 +13,11 @@ else()
 endif()
 
 project(quicr
-        VERSION 1.0.0
+        VERSION 1.0.1
         DESCRIPTION "QuicR, a Media over QUIC Library"
         LANGUAGES CXX)
 
 configure_file( src/version.h.in ${CMAKE_BINARY_DIR}/include/quicr/version.h )
```

Comment:
`${CMAKE_BINARY_DIR}` I think is going to be the top level build directory of whatever project is the root (which might not be libquicr). This still works because the the corresponding `target_include_directories` matches it, but it might not be what we want. We might want to be using `CMAKE_CURRENT_BINARY_DIR` instead, to keep it under libquicr. 

For example if you built a containing project (e.g laps), this would put the version header at `/someProject/build/include/quicr/version.h` not `someProject/build/dependencies/libquicr/include/quicr/version.h`